docs/deployment/k8s/index.mdx

```diff
@@ -37,172 +37,123 @@ If you are running Kubernetes locally, the following steps will help you start a
 Make sure you have a Kubernetes cluster running locally. Some good options include [minikube](https://minikube.sigs.k8s.io/docs/), [kind](https://kind.sigs.k8s.io/), [k3s](https://k3s.io/), or [k0s](https://k0sproject.io/).
 :::
 
-#### Deploying NATS
+#### Download the operator
 
-Add the NATS Helm repo:
+Download the operator from GitHub and navigate to the `quickstart` subfolder:
 
 ```shell
-helm repo add nats https://nats-io.github.io/k8s/helm/charts/
+git clone https://github.com/wasmCloud/wasmcloud-operator.git
 ```
-
-Install the upstream NATS Helm chart to start a cluster with the following
-`values.yaml` file:
-
-```yaml
-config:
-  cluster:
-    enabled: true
-    replicas: 3
-  leafnodes:
-    enabled: true
-  jetstream:
-    enabled: true
-    fileStore:
-      pvc:
-        size: 5Gi
-    merge:
-      domain: default
-  websocket:
-    enabled: true
-    port: 4223
-```
-
 ```shell
-helm upgrade --install -f values.yaml nats-cluster nats/nats
+cd examples/quickstart
 ```
 
-#### Deploying wadm
+#### Deploy NATS
 
-You can deploy wadm to your Kubernetes cluster with a Helm chart hosted by wasmCloud. Use this `values.yaml` file to define a minimal deployment that connects with the NATS server deployed above: 
+Add the NATS Helm repo:
 
-```yaml
-wadm:
-  config:
-    nats:
-      server: "nats-cluster.default.svc.cluster.local:4222"
+```shell
+helm repo add nats https://nats-io.github.io/k8s/helm/charts/
 ```
-
-Run `helm install`:
+Install the upstream NATS Helm chart to start a cluster with the `values.yaml` file from the quickstart folder:
 
 ```shell
-helm install wadm -f values.yaml oci://ghcr.io/wasmcloud/charts/wadm
+helm upgrade --install -f nats-values.yaml nats nats/nats

```

Comment:
In order to no longer having to git checkout the operator repository, we can instead reference the quick start values files via URL:
```suggestion
helm upgrade --install -f https://raw.githubusercontent.com/wasmCloud/wasmcloud-operator/main/examples/quickstart/nats-values.yaml nats nats/nats
```